### PR TITLE
Add panel and graph

### PR DESCRIPTION
**Pull request by iamdiluxedbutcooler**
#150

Screenshot 2567-11-06 at 19 21 20

### CODE REVIEW

**Review by ZShunRen — APPROVED** [review]
LGTM

File: src/main/resources/view/css/DarkTheme.css [hunk 1]
```diff
@@ -406,6 +407,30 @@
...
+}
+
+#commandTypeLabel {
+    -fx-font-size: 11px;
+    -fx-text-fill: #1C1C1C;
+}
+
+#filterField, #clientListPanel, #clientWebpage {
+    -fx-effect: innershadow(gaussian, black, 10, 0, 0, 0);
+}
```
Comment (itsme-zeix) [inline src/main/resources/view/css/DarkTheme.css:432]: Where do the shadows show up? (The ss could be outdated, idk)

File: src/main/resources/view/css/StatusPieChart.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    -fx-background-color: #1C1C1C;
+    -fx-padding: 2;
+}
+
+/* Chart title styles */
+.chart-title {
+    -fx-text-fill: white;
+    -fx-font-size: 16px;
+    -fx-padding: 0 0 -2 0;
+}
```
Comment (itsme-zeix) [inline src/main/resources/view/css/StatusPieChart.css:12]: Could consider using a bolded font instead of the uppercase letters.

**Review by itsme-zeix — APPROVED** [review]
Amazing work all around! Looks really, really good!

**Pull request merged**